rct229/rulesets/ashrae9012019/section19/section19rule13.py

```diff
@@ -0,0 +1,255 @@
+                    )
+                    design_cooling_supply_air_temp_setpoint = getattr_(
+                        terminal_b,
+                        "Terminal",
+                        "supply_design_cooling_setpoint_temperature",
+                    )
+
+                    if zone_has_lab_space:
+                        zone_info[zone_id_b][
+                            "all_design_setpoints_delta_Ts_are_per_reqs"
+                        ] = (
+                            True
+                            if (
+                                design_heating_supply_air_temp_setpoint
+                                - design_cooling_supply_air_temp_setpoint
+                            )
+                            != TEMP_DELTA_17F
+                            else False
+                        )
+
+                    else:
+                        zone_info[zone_id_b][
+                            "all_design_setpoints_delta_Ts_are_per_reqs"
+                        ] = (
+                            True
+                            if (
+                                design_heating_supply_air_temp_setpoint
+                                - design_cooling_supply_air_temp_setpoint
+                            )
+                            != TEMP_DELTA_20F
+                            else False
+                        )
+
+        return {
+            "zone_info": zone_info,
+        }
+
+    def is_applicable(self, context, data=None):
+        rmi_b = context.baseline
+        baseline_system_types_dict = get_baseline_system_types(rmi_b)
+
+        return any(
+            [
+                baseline_system_type_compare(system_type, applicable_sys_type, False)
+                for system_type in baseline_system_types_dict.keys()
+                for applicable_sys_type in APPLICABLE_SYS_TYPES
+            ]
+        )
+
+    class HVACRule(RuleDefinitionBase):
+        def __init__(self):
+            super(Section19Rule13.HVACRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, True, True),
+                required_fields={
+                    "$": ["fan_system"],
+                    "fan_system": ["minimum_outdoor_airflow"],
+                },
+            )
+
+        def get_calc_vals(self, context, data=None):
+            hvac_b = context.baseline
+            hvac_id_b = hvac_b["id"]
+            zone_info = data["zone_info"][hvac_id_b]
+            all_design_setpoints_delta_Ts_are_per_reqs_b = zone_info[
+                "all_design_setpoints_delta_Ts_are_per_reqs"
+            ]
+            are_all_hvac_sys_fan_objs_autosized_b = zone_info[
+                "are_all_hvac_sys_fan_objs_autosized"
+            ]
+
+            fan_sys_b = hvac_b["fan_system"]
+            supply_fans_airflow_b = (
+                get_fan_system_object_supply_return_exhaust_relief_total_power_flow(
+                    fan_sys_b
+                )
+            )
```

Comment:
We should either update the variable name to:
`fans_total_power_flow_b` or get the supply fans calculated values from the return value:
```python
fans_total_power_flow_b = get_fan_system_object_supply_return_exhaust_relief_total_power_flow(fan_sys_b)
supply_fan_qty = fans_total_power_flow_b["supply_fans_qty"]
``` 

For reporting purpose, I prefer the second approach which avoid having a nested dict in the calc_val value.

Reply:
Addressed.